testsuite/MDAnalysisTests/utils/test_selections.py

```diff
@@ -176,20 +180,21 @@ def _assert_selectionstring(self):
                      err_msg="PyMOL selection was not written correctly")
 
 
-
 def spt2array(line):
     """Get name of and convert Jmol SPT definition to integer array"""
     match = re.search(r'\@~(\w+) \(\{([\d\s]*)\}\)', line)
     return match.group(1), np.array(match.group(2).split(), dtype=int)
 
 
 class TestSelectionWriter_Jmol(_SelectionWriter):
+    __test__ = True
+
     writer = MDAnalysis.selections.jmol.SelectionWriter
     filename = "CA.spt"
     ref_name, ref_indices = spt2array(
-        ( '@~ca ({4 21 45 64 83 102 121 128 140 152 159 169 176 198 205 219 236'
-          ' 246 263 283 302 319 334 356});')
```

Comment:
Yes and we try to satisfy PEP8 in the code we touch and not the whole file. It's noise and in an old code base not everything is perfect following a coding style